README.md

```diff
@@ -371,8 +371,8 @@ not required as this software includes P4C generated artifacts.
   Get the IP address assigned to one of the pods using ifconfig. Then, ping that
   address from the other pod.
   ```bash
-  # kubectl exec test-pod2 -- ifconfig eth0
-  # kubectl exec test-pod -- ping 10.244.0.6
+  # kubectl exec test-pod -- ifconfig eth0
```

Comment:
This change is not needed. The only thing required is to get IP address of one pod and ping it from the other pod. Which pod to choose as the first one and which one as the other one, does not make any difference.